helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -187,6 +187,7 @@ splunk_hec/platform_logs:
   timeout: {{ .Values.splunkPlatform.timeout }}
   splunk_app_name: {{ .Chart.Name }}
   splunk_app_version: {{ .Chart.Version }}
+  profiling_data_enabled: false
```

Comment:
This would prevent users from falling back to an older collector version if needed for some reason. May be worth planning a ~feature set helper in the future to make this easier.

Reply:
We can add a condition based `{{ .Chart.AppVersion }}` We haven't been doing that so far, but maybe makes sense here

Reply:
Hmm. actually I don't think `.Chart.AppVersion` can be set by the end user

Reply:
Probably we can parse `image.otelcol.tag`, but not sure it's worth the added complication. I'd rather keep it as is and add such backward compatibility if there is an ask for that